src/tlo/util.py

```diff
@@ -118,11 +118,14 @@ def sample_outcome(probs: pd.DataFrame, rng: np.random.RandomState):
     return outcome.loc[outcome != '_'].to_dict()
 
 
+BitsetDType = Property.PANDAS_TYPE_MAP[Types.BITSET]
```

Comment:
I don't think we want to add a type hint of the form suggested for `BitsetDType` here as that would then mean we are specifying the concrete type at two points in the code so would need to keep these in sync. `BitsetDType` itself is definitely not defaulting to `Any` - the type checker might not be able to infer a more precise type for it, but the variable itself does map to `np.uint32`. We could add an explicit type hint to the `Property.PANDAS_TYPE_MAP` dict though I think currently this would need to be `Dict[Types, Any]` anyway as there is no more specific type that covers all the values.

Reply:
No worries - I just noticed that VSCode was always flagging `BitsetDType` as `Any` whilst editing, in spite of the explicit assignment here.

For the record, it seems that VSCode only follows typehints and doesn't do evaluation:

```python
BitsetDType = Property.PANDAS_TYPE_MAP[Types.BITSET]
            = Dict[Types, Any][evaluated with an instance of Types]
            = Any
```

hence why `BitsetDType` is flagged as `Any`.